app/assets/javascripts/mapknitter/Map.js

```diff
@@ -88,23 +88,30 @@ MapKnitter.Map = MapKnitter.Class.extend({
 
           });
 
-          var corners = [ 
-                new L.latLng(warpable.nodes[0].lat,
-                             warpable.nodes[0].lon),
-                new L.latLng(warpable.nodes[1].lat,
-                             warpable.nodes[1].lon),
-                new L.latLng(warpable.nodes[3].lat,
-                             warpable.nodes[3].lon),
-                new L.latLng(warpable.nodes[2].lat,
-                             warpable.nodes[2].lon)
+          var corners = [
+            L.latLng(warpable.nodes[0].lat,
+              warpable.nodes[0].lon),
+            L.latLng(warpable.nodes[1].lat,
+              warpable.nodes[1].lon),
+            L.latLng(warpable.nodes[3].lat,
+              warpable.nodes[3].lon),
+            L.latLng(warpable.nodes[2].lat,
+              warpable.nodes[2].lon)
           ];
 
-          var img = new L.DistortableImageOverlay(
+          var img = L.distortableImageOverlay(
             warpable.srcmedium,
-            { 
+            {
               keymapper: false,
+              actions: this.imgActionArray(),
```

Comment:
Here's where we're seeing the issue. I think it's because we're using `this` but the scope may need to be tracked as this is a constructor inside a constructor...